### ISSUE TITLE
Use GlossHook. 

### ISSUE THREAD

**Issue by XMDS** [beloncode/SAMobile#1]
It's basically stable
https://github.com/XMDS/GlossHook/releases/tag/v1.0

**Comment by shadergz** [comment]
Thank I has been think in makes this change, but my implementation works as expected by now - as the project grow up i will later change it

**Comment by XMDS** [comment]
@beloncode What is your end goal?  samp Android?

**Comment by shadergz** [comment]
Yeah, or something near of build69 by 4x11 was a day! It's my funy hobby project, i'm working by few months in other projects on my organization (in a emulator for Android devices) and i work daily in a software developemnt companny. but i will develop this project when i miss fault of samp for Android 🙃

**Comment by XMDS** [comment]
@beloncode  Do you have discord?

**Comment by shadergz** [comment]
Yea checkout my profile description but I dont have any specific group

**Comment by XMDS** [comment]
@beloncode  I can't seem to add your discord account as a friend with the info on your git, when I click the link it opens discord and nothing happens

**Comment by shadergz** [comment]
> @beloncode  I can't seem to add your discord account as a friend with the info on your git, when I click the link it opens discord and nothing happens

I will take a look

**Comment by XMDS** [comment]
@beloncode  Hello, if you have time, I think you can use GlossHook, test it in your project, the latest 1.5.1 is already very stable, and it has almost any hook api you need, and it has multiple calls to the same location  Hook compatibility is very good.  They are currently ready to be used in the latest AML and ARMPatch.

https://github.com/XMDS/GlossHook/releases/tag/v1.5.1

Actually, I need to test it in many projects

**Comment by shadergz** [comment]
> @beloncode  Hello, if you have time, I think you can use GlossHook, test it in your project, the latest 1.5.1 is already very stable, and it has almost any hook api you need, and it has multiple calls to the same location  Hook compatibility is very good.  They are currently ready to be used in the latest AML and ARMPatch.
> 
> https://github.com/XMDS/GlossHook/releases/tag/v1.5.1
> 
> Actually, I need to test it in many projects

In the future, I will create a branch and try to use your hook within this project. But for now, it's time to proceed with the local player objects
